docs/source/pillars/information_governance.md

```diff
@@ -86,6 +86,12 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
 * - You report on and share outcomes of each audit of your TRE organisation with the required bodies
   - This may included regulatory bodies or the organisations that manage accreditations you have
   - Mandatory
+* - You log user actions, network connection attempts, infrastructure actions and data access within a TRE
+  - The greater the number of (and diversity of) actions logged, the higher the chance that any bad behaviours, intentional or unintentional, are spotted.
```

Comment:
Do you think that is true?

If you have more logs does it make it easier to find what you are looking for?

In any cases, the logs are no use unless you look at them.

Reply:
I have clarified this - I meant greater coverage of what is logged